doc/api/assert.markdown

```diff
@@ -89,8 +89,9 @@ parameter is undefined, a default error message is assigned.
 
 ## assert.deepStrictEqual(actual, expected[, message])
 
-Generally identical to [`assert.deepEqual()`][] with the exception that primitive
-values are compared using the strict equality operator ( `===` ).
+Generally identical to [`assert.deepEqual()`][] with the exceptions that
+primitive values are compared using the strict equality operator ( `===` ), and
+object prototypes are included in the comparison.
```

Comment:
it would have been better if we could somehow say that prototypes should be the same. But yeah the proposed change also is okay.